Detectors/gconfig/g4config.in

```diff
@@ -14,6 +14,7 @@
 /mcDet/setIsMaxStepInLowDensityMaterials true
 /mcDet/setMaxStepInLowDensityMaterials 10 m
 /mcMagField/setConstDistance 1 mm
+/mcDet/setIsZeroMagField true
```

Comment:
ok for me. I would have found "enableZeroMagFieldMedia" or something a bit more clear. The present could possibly be misinterpreted as setting the field to zero.